src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <!--Fonts-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <!--Style-->
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div class="card page__card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        class="card__image"
+      >
+
+      <h1 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
it's better use h2 since we use h1 only once at our page (usually for titles)
```suggestion
      <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```